pystats_docs.md

```diff
@@ -0,0 +1,81 @@
+# pystats documentation
+
+This describes the meaning of the various sections and fields in the pystats output.
+
+## Execution counts
+
+This is the count of how many times each Tier 1 instruction is executed.
+
+The "miss ratio" column shows the percentage of times the instruction executed that it deoptimized.
+
+## Pair counts
+
+Shows the Top 100 most frequently occurring pairs of instructions as executed.
+
+## Predecessor / Successor pairs
+
+Shows the top 5 instructions that are executed before and after each instruction.
+
+## Specialization stats
+
+TBD
+
+## Specialization effectiveness
+
+TBD
+
+## Call stats
+
+TBD
```

Comment:
This is shows what fraction of calls to Python functions are inlined (no call at the C level) and for those that are not, where the call comes from. The various categories overlap.

Also includes count of frame objects created.

Read the code for more details.